advocacy_docs/partner_docs/EsriArcGISEnterprise/04-ConfiguringEsriArcGISEnterprise.mdx

```diff
@@ -0,0 +1,184 @@
+NOTICE:  SRID value -1 converted to the officially unknown SRID value 0
+INSERT 0 1
+postgis=# INSERT INTO ROADS (ID,GEOM,NAME ) VALUES (5,ST_GeomFromText('LINESTRING(0 10,0 0)',-1),'Main Street');
+NOTICE:  SRID value -1 converted to the officially unknown SRID value 0
+INSERT 0 1
+postgis=# INSERT INTO ROADS (ID,GEOM,NAME ) VALUES (6,ST_GeomFromText('LINESTRING(10 0,0 0)',-1),'Lipton Street');
+NOTICE:  SRID value -1 converted to the officially unknown SRID value 0
+INSERT 0 1
+postgis=#
+```
+
+7. Show the data that was added in table format
+
+```bash
+postgis=#
+postgis=# SELECT id, ST_AsText(geom) AS geom, name FROM ROADS order by id;
+ id |         geom          |      name
+----+-----------------------+----------------
+  1 | LINESTRING(0 10,0 0)  | Beacon Road
+  2 | LINESTRING(0 0,0 10)  | Violet Road
+  3 | LINESTRING(0 0,10 0)  | Skelton Street
+  4 | LINESTRING(0 0,10 10) | Fifth Avenue
+  5 | LINESTRING(0 10,0 0)  | Main Street
+  6 | LINESTRING(10 0,0 0)  | Lipton Street
+(6 rows)
+
+postgis=#
+
+```
+
+8. Show `PostGIS` version
+
+```bash 
+postgis=# SELECT PostGIS_Version();
+            postgis_version
+---------------------------------------
+ 3.2 USE_GEOS=1 USE_PROJ=1 USE_STATS=1
+(1 row)
+
+```
+### GIS Data
+
+Users can either create their own GIS datasets or they can download GIS data if needed.
+
+The test cases in this guide are using the sample 'HongKong GDB' data file for demonstration, which can be found here: https://www.arcgis.com/home/item.html?id=96a56c14f72c4638961f4ee9a271e34d.
+
+## Configuring Esri ArcGIS Pro
+To share maps and data from Esri ArcGIS Pro to an Esri ArcGIS Enterprise Environment you will need to have Esri ArcGIS Pro installed and active on your machine with some maps, data or Enterprise Geodatabases present to share.
+
+This guide does not go over installing and configuring Esri ArcGIS Pro, but those steps can be found in the Esri ArcGIS Pro EDB Implementation guide.
+
+## Configuring Esri ArcGIS Enterprise Environment
+
+Esri ArcGIS Enterprise Environment must be configured and running. Installing and configuring it is not part of this Implementation Guide, for more information please refer to the following link:
+https://enterprise.arcgis.com/en/get-started/latest/windows/tutorial-creating-your-first-web-gis-configuration.htm
+
+## Esri ArcGIS Pro Configuration for Enterprise Sharing
+
+Before you can share from your Esri ArcGIS Pro environment to an Enterprise Portal that you have created, you will need to add that Portal in your Esri ArcGIS Pro environment.  You will need to be signed in to your Esri ArcGIS Enterprise Portal to share maps and data to your Esri ArcGIS Enterprise Environment.
+
+1. Click `Projects` then select `Portals`.
+ 2. Click `Add Portal` to add the URL of the Portal you wish to use.
+ 3. Set the Portal as the active portal by selecting the option from the three dots to the right.
+ 4. Right click the Portal and select `Sign in`.
```

Comment:
Should this be next?

After you have signed in to your Esri ArcGIS Enterprise Portal you will then be ready to share to an Esri ArcGIS Enterprise environment.

Reply:
Added